### PR TITLE
document testing of hadoopfs

### LINKED ISSUE #3051: Documentation guarantees of lakeFSFS support for a Hadoop version
Document (we already know):

- Esti tests that check how HadoopFS pass for different Spark and Hadoop aws versions.
- FS contract tests pass for the supported Hadoop versions.

Challenge: We will need to publish multiple Hadoop file systems, we may be
able to shade in hadoop-aws to avoid conflicts.

### PR DESCRIPTION
**Pull request by johnnyaug**
Resolves #3051.

### CODE REVIEW

File: docs/integrations/spark.md [hunk 1]
```diff
@@ -185,14 +185,12 @@ df.write.partitionBy("example-column").parquet(s"s3a://${repo}/${branch}/output-
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes or revert them.
 
-## Use the lakeFS-specific Hadoop FileSystem
+## Use the lakeFS Hadoop FileSystem
```
Comment (arielshaqed) [inline docs/integrations/spark.md:188]: # 🎆 🎇 🥇 😀

File: docs/integrations/spark.md [hunk 2]
```diff
@@ -337,6 +334,11 @@ df.write.partitionBy("example-column").parquet(s"lakefs://${repo}/${branch}/outp
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes or revert them.
 
+### Notes for the lakeFS Hadoop FileSystem
+
+* Since data will not be sent to the lakeFS server, using this mode maximizes data security.
+* The FileSystem implementation is tested with the latest Spark 2.X and Spark 3.X Bitnami images.
```
Comment (arielshaqed) [inline docs/integrations/spark.md:340]: Can we state that one of these is Hadoop 2 and the other with Hadoop 3?

**Review by arielshaqed — APPROVED** [review]
Thanks!

File: docs/integrations/spark.md (hunk 2 above)
Comment (johnnyaug) [reply docs/integrations/spark.md]: Done

**Pull request merged**